pkg/apis/provisioning.cattle.io/v1/cluster.go

```diff
@@ -28,6 +28,8 @@ type ClusterSpec struct {
 	DefaultPodSecurityPolicyTemplateName string          `json:"defaultPodSecurityPolicyTemplateName,omitempty" norman:"type=reference[podSecurityPolicyTemplate]"`
 	DefaultClusterRoleForProjectMembers  string          `json:"defaultClusterRoleForProjectMembers,omitempty" norman:"type=reference[roleTemplate]"`
 	EnableNetworkPolicy                  *bool           `json:"enableNetworkPolicy,omitempty" norman:"default=false"`
+
+	RedeploySystemAgentGeneration int64 `json:"redeploySystemAgentGeneration,omitempty"`
```

Comment:
Alignment

Reply:
This actually passed go fmt.

Reply:
Yeah, didn't notice a newline - it's in own block so doesn't have to align.